### PR TITLE
TINY-8323: release notes 5.10.3

### PR DESCRIPTION
**Pull request by EkimChau**
Related Ticket: 

Description of Changes:
* Placeholder text

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: release-notes/release-notes5103.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+{{site.productname}} 5.10.3 provides fixes for the following bugs:
+
+- Fixed changing the alignment of an inline image removes the alignment of the parent.
+- Fixed when toggling back from fullscreen mode, the scrollbar position is incorrect.
+
+## Security fixes
+
+{{site.productname}} 5.10.3 provides fixes for the following security issues:
+
+All 3 server-side components have been updated to include dependency updates addressing security issues. This includes High severity vulnerability fixes.
```
Comment (lnewson) [inline release-notes/release-notes5103.md:96]: @tjdett can you please double check this? I believe the issues were only Low severity IIRC.

File: release-notes/release-notes5103.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+- Deploying the server-side components, see: [Server-side component installation]({{site.baseurl}}/enterprise/server/).
+- Deploying the server-side components using Docker, see: [Containerized service deployments]({{site.baseurl}}/enterprise/server/dockerservices/).
+
+## General bug fixes
+
+{{site.productname}} 5.10.3 provides fixes for the following bugs:
+
+- Fixed changing the alignment of an inline image removes the alignment of the parent.
+- Fixed when toggling back from fullscreen mode, the scrollbar position is incorrect.
```
Comment (lnewson) [inline release-notes/release-notes5103.md:90]: These have grammar issues and aren't what was originally in the changelog, so let's use what was there originally instead as that's what we normally do (unless it needs minor rewording):
```suggestion
- Alignment would sometimes be removed on parent elements when changing alignment on certain inline nodes, such as images.
- The `fullscreen` plugin would incorrectly reset the scroll position when exiting fullscreen mode.
```

File: _data/nav.yml [hunk 3]
```diff
@@ -967,6 +967,14 @@
...
   pages:
   - url: "6.0-upcoming-changes"
+  - url: "release-notes5103"
+    pages:
+    - url: "#Overview"
+    - url: "#Accompanying Premium self-hosted server-side component changes"
+    - url: "#General bug fixes"
+    - url: "#Security fixes"
+    - url: "#Upgrading to the latest version of TinyMCE 5"
+
```
Comment (lnewson) [inline _data/nav.yml:977]:
```suggestion
```
There shouldn't be a new line here.

**Review by lnewson — CHANGES_REQUESTED** [review]
https://github.com/tinymce/tinymce-docs/blob/main/.api-version this file also needs updating to use 5.10.3.

- [x] Update api version

File: _data/nav.yml (hunk 3 above)
Comment (EkimChau) [reply _data/nav.yml:977]: fixed

File: release-notes/release-notes5103.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+- [Accompanying Premium self-hosted server-side component changes](#accompanyingpremiumself-hostedserver-sidecomponentchanges)
+- [General bug fixes](#generalbugfixes)
+- [Security fixes](#securityfixes)
+- [Upgrading to the latest version of TinyMCE 5](#upgradingtothelatestversionoftinymce5)
+
+{{site.releasenotes_for_stable}}
+
+## Accompanying Premium self-hosted server-side component changes
+
+The {{site.productname}} 5103 release includes accompanying changes affecting the {{site.productname}} **self-hosted** services for the following plugins:
```
Comment (lnewson) [inline release-notes/release-notes5103.md:22]:
```suggestion
The {{site.productname}} 5.10.3 release includes accompanying changes affecting the {{site.productname}} **self-hosted** services for the following plugins:
```

File: release-notes/release-notes5103.md (hunk 2 above)
Comment (EkimChau) [reply release-notes/release-notes5103.md]: fixed

File: release-notes/release-notes5103.md (hunk 1 above)
Comment (EkimChau) [reply release-notes/release-notes5103.md]: i've updated to Medium to match this report https://nvd.nist.gov/vuln/detail/CVE-2021-42550
happy to change

File: release-notes/release-notes5103.md (hunk 2 above)
Comment (lnewson) [reply release-notes/release-notes5103.md]: Not fixed... you've still changed them and they have grammar issues. We also shouldn't start it with fixed, as the section heading/intro already mentions they are fixes.

File: release-notes/release-notes5103.md (hunk 1 above)
Comment (EkimChau) [reply release-notes/release-notes5103.md]: have confirmed with tjdett, is Low

**Review by ltrouton — COMMENTED** [review]
I believe we also need to change the `.api-version` file from looking at the previous release notes PR?

File: release-notes/release-notes5103.md (hunk 2 above)
Comment (lnewson) [reply release-notes/release-notes5103.md]: Have fixed this myself as mentioned in slack.

**Comment by lnewson** [comment]
Have fixed the `.api-version` file as well, as other fixes so I think this should be good now.

File: release-notes/release-notes5103.md (hunk 2 above)
Comment (brian-forte) [reply release-notes/release-notes5103.md]: editorial suggestion from the new guy:

\+ {{site.productname}} 5.10.3 fixes the following bugs:

\+ - in some circumstances (for example, with images), changing inline node alignments removed the parent element’s alignment.

\+ - the `fullscreen` plugin incorrectly reset the scroll position when exiting fullscreen mode.

File: release-notes/release-notes5103.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+- [Accompanying Premium self-hosted server-side component changes](#accompanyingpremiumself-hostedserver-sidecomponentchanges)
+- [General bug fixes](#generalbugfixes)
+- [Security fixes](#securityfixes)
+- [Upgrading to the latest version of TinyMCE 5](#upgradingtothelatestversionoftinymce5)
+
+{{site.releasenotes_for_stable}}
+
+## Accompanying Premium self-hosted server-side component changes
+
+The {{site.productname}} 5.10.3 release includes accompanying changes affecting the {{site.productname}} **self-hosted** services for the following plugins:
```
Comment (brian-forte) [inline release-notes/release-notes5103.md:22]: Suggested tweak:

\+ The {{site.productname}} 5.10.3 release includes changes affecting the {{site.productname}} **self-hosted** services for the following plugins:

File: release-notes/release-notes5103.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+## General bug fixes
+
+{{site.productname}} 5.10.3 provides fixes for the following bugs:
+
+- Fixed alignment would sometimes be removed on parent elements when changing alignment on certain inline nodes, such as images.
+- Fixed the `fullscreen` plugin would incorrectly reset the scroll position when exiting fullscreen mode.
+
+## Security fixes
+
+{{site.productname}} 5.10.3 provides fixes for the following security issues:
```
Comment (brian-forte) [inline release-notes/release-notes5103.md:74]: suggested tweak:

\+ {{site.productname}} 5.10.3 fixes the following security issues:

\+ All 3 server-side components updates include dependency updates that address security issues. This includes Low severity vulnerability fixes.

File: release-notes/release-notes5103.md (hunk 5 above)
Comment (lnewson) [reply release-notes/release-notes5103.md:22]: This is a template thing, so if we're changing this we need to change others: https://github.com/tinymce/tinymce-docs/blob/main/_new_content_templates/blank_releasenotes.md#accompanying-premium-self-hosted-server-side-component-changes

File: _new_content_templates/blank_releasenotes.md [hunk 7]
```diff
@@ -120,7 +120,7 @@ For information on using premium skins and icon packs, see: [Premium Skins and I
 
 ## Accompanying Premium self-hosted server-side component changes
 
-The {{site.productname}} vnumtxt release includes accompanying changes affecting the {{site.productname}} **self-hosted** services for the following plugins:
+The {{site.productname}} vnumtxt release includes changes affecting the {{site.productname}} **self-hosted** services for the following plugins:
```
Comment (metricjs) [inline _new_content_templates/blank_releasenotes.md:123]: `vnumtxt`?

Comment (metricjs) [reply _new_content_templates/blank_releasenotes.md:123]: Oh wait, this is a template. Can probably ignore?

Comment (lnewson) [reply _new_content_templates/blank_releasenotes.md:123]: @metricjs that's the template file, so that's expected.

**Review by ltrouton — APPROVED** [review]

**Review by metricjs — APPROVED** [review]

**Pull request merged**